packages/terra-tabs/src/terra-dev-site/test/tabs/Tabs/AdditionalTabs.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Tabs from 'terra-tabs';
+import TabContent from './TabContentTemplate';
+
+class AdditionalTabsTest extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tabKeys: [...Array(5).keys()],
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({
+        tabKeys: [...Array(20).keys()],
+      });
+    }, 3000);
```

Comment:
Could we test this without adding a timeout? Maybe utilize a button to add tabs? Or at least try to minimize the timeout to minimize the time added to the test run?

Reply:
Done.